ProjectLighthouse.Servers.GameServer/Controllers/Resources/ResourcesController.cs

```diff
@@ -42,11 +42,11 @@ public IActionResult GetResource(string hash)
         string fullPath = Path.GetFullPath(path);
 
         // Prevent directory traversal attacks
-        if (!fullPath.StartsWith(FileHelper.FullResourcePath)) return this.BadRequest();
+        if (!fullPath.StartsWith(FileHelper.FullResourcePath)) return this.StatusCode(400);
 
         if (FileHelper.ResourceExists(hash)) return this.File(IOFile.OpenRead(path), "application/octet-stream");
 
-        return this.NotFound();
+        return this.StatusCode(404);
```

Comment:
Reason for these? Just curious.

Reply:
I agree with koko, did these not work before